lib/ansible/modules/storage/netapp/sf_check_connections.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python
+
+# (c) 2017, NetApp, Inc
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+DOCUMENTATION = '''
+
+module: sf_check_connections
+
+short_description: Check connectivity to MVIP and SVIP.
+extends_documentation_fragment:
+    - netapp.solidfire
+version_added: '2.3'
+author: Sumit Kumar (sumit4@netapp.com)
+description:
+- Used to test the management connection to the cluster.
+- The test pings the MVIP and SVIP, and executes a simple API method to verify connectivity.
+
+options:
+
+  skip:
+    required: false
+    description:
+    - Skip checking connection to SVIP or MVIP.
+    choices: ['svip', 'mvip']
+    default: None
+
+  mvip:
+    required: false
+    description:
+    - Optionally, use to test connection of a different MVIP.
+    - This is not needed to test the connection to the target cluster.
+    default: None
+
+  svip:
+    required: false
+    description:
+    - Optionally, use to test connection of a different SVIP.
+    - This is not needed to test the connection to the target cluster.
+    default: None
+
+'''
+
+
+EXAMPLES = """
+   - name: Check connections to MVIP and SVIP
+     sf_check_connections:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+"""
+
+RETURN = """
+msg:
```

Comment:
It is not in the correct format and only documents common returns, which must not be documented. Please remove this return content, but keep the variable, e.g.
~~~ yaml
RETURN = """
"""
~~~